test/extensions/health_checkers/redis/redis_test.cc

```diff
@@ -53,6 +53,28 @@ class RedisHealthCheckerTest
                                Upstream::HealthCheckEventLoggerPtr(event_logger_), *this));
   }
 
+  void setupAlwaysLogHealthCheckFailures() {
```

Comment:
Do this redis tests here and in the other file provide any additional real coverage? Isn't all the new functionality in health checker base? Just wondering if there is reason to add these new tests.

Reply:
The redis is a custom health check, having these tests would make sure it works correctly and in future refactor it doesn't override the behavior from the health check base.  Also, health check failures for stateful services like redis has higher impact than stateless services, so I really want to make sure this scenario works.

Reply:
In general I don't think this are great reasons to add a bunch of new tests that don't add any actual coverage, but since you already wrote them we can keep them.